doc/classes/String.xml

```diff
@@ -662,16 +662,17 @@
 		<method name="rsplit">
 			<return type="PoolStringArray">
 			</return>
-			<argument index="0" name="divisor" type="String">
+			<argument index="0" name="delimiter" type="String">
 			</argument>
 			<argument index="1" name="allow_empty" type="bool" default="True">
 			</argument>
 			<argument index="2" name="maxsplit" type="int" default="0">
 			</argument>
 			<description>
-				Splits the string by a [code]divisor[/code] string and returns an array of the substrings, starting from right.
-				[b]Example:[/b] [code]"One,Two,Three"[/code] will return [code]["One","Two","Three"][/code] if split by [code]","[/code].
-				If [code]maxsplit[/code] is specified, then it is number of splits to do, default is 0 which splits all the items.
+				Splits the string by a [code]delimiter[/code] string and returns an array of the substrings, starting from right.
+				The splits in the returned array are sorted in the same order as the original string, from left to right.
+				If [code]maxsplit[/code] is specified, it defines the number of splits to do from the right up to [code]maxsplit[/code]. The default value of 0 means that all items are split, thus giving the same result as [method split].
+				[b]Example:[/b] [code]"One,Two,Three,Four"[/code] will return [code]["Three","Four"][/code] if split by [code]","[/code] with maxsplit of [code]"2"[/code].
```

Comment:
As mentioned previously, the integer value `2` should not be within double quotes, which are the text representation of Strings. So it should be `with a [code]maxsplit[/code] value of 2`.

It would also be great if you could do similar changes to the `split` description for consistency.

Otherwise the changes are great, thanks a lot for putting up with the cumbersome Git workflow. You'll see that it gets easier :)

Reply:
> As mentioned previously, the integer value 2 should not be within double quotes, which are the text representation of Strings. So it should be with a [code]maxsplit[/code] value of 2.

So values should not be within [code][/code] tags? Should i also remove the tags from [code]","[/code] since its a value for the delimiter?

> It would also be great if you could do similar changes to the split description for consistency.

Sure!

> Otherwise the changes are great, thanks a lot for putting up with the cumbersome Git workflow. You'll see that it gets easier :)

Thank you! I think i'm getting the hang of it now that i switched to cmd. Managed to remove the commits from master (#29430) and close it since it was a duplicate to this one. I tend to make a mess when i learn and thats mostly ok if i'm by myself but now i felt i'm messing with other peoples work also.

Reply:
Don't worry about messing up other people's work. The absolute worst case scenario is you mess up your copy of other people's work. But it seems that you have it figured out now.

Reply:
> So values should not be within [code][/code] tags? Should i also remove the tags from [code]","[/code] since its a value for the delimiter? Or just remove quotes from the integer value?

I don't have a strong opinion on that one, I guess you can use either `2` or `[code]2[/code]`. To me the latter emphasizes that it's the programming value "2" which represents an integer, while the former would be the "usual" number 2 that we refer to in plain English. (Don't know if that makes sense...)